Abstracts/Events/Dispatcher/Dispatcher.php

```diff
@@ -14,18 +14,34 @@
  */
 class Dispatcher extends EventDispatcher
 {
-    public function dispatch($event, $payload = [], $halt = false)
-    {
-        if ($event instanceof ShouldHandle) {
-            $job = new EventJob($event);
-            $job = isset($event->jobDelay) ? $job->delay($event->jobDelay) : $job;
-            $job = isset($event->jobQueue) ? $job->onQueue($event->jobQueue) : $job;
-            new JobDispatcher($job);
-        } else if ($event instanceof ShouldHandleNow) {
-            $event->handle();
-        }
-
-        return parent::dispatch($event, $payload, $halt);
+  public function dispatch($event, $payload = [], $halt = false)
```

Comment:
can you please fix indentation from 2 spaces to 4 spaces throughout your changes? thank you ;)